packages/core/src/state.ts

```diff
@@ -1,10 +1,11 @@
 import { Observable, Subject, noop, ReplaySubject, Subscription, identity } from 'rxjs'
 import { Reducer } from 'react'
```

Comment:
Not included in this PR, but seems we should avoid importing stuff from any frameworks in @sigi/core

Reply:
Just a type, will be striped in runtime